molecule/legacy/playbook.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Converge
+  hosts: all
+  pre_tasks:
+    - name: update package manager
+      when: "ansible_pkg_mgr == 'apt'"
+      apt:
+        update_cache: true
+  roles:
+    - role: ansible-role-clang
+      vars:
+        clang_versions:
+          - 5
```

Comment:
@Teebor-Choka why introduce this scenario? The default scenario could just test a big list of versions, and then the role could document which versions are actually supported. Also, I think it would be a good idea to test out all the intermediate versions, since the packages vary from version to version.

Reply:
I'll remove multiple version support from this role, it is confusing and obviously does not work the way one would expect.